### PR TITLE
Added price next to each items in sack display, Added runes/gemstones sack display

### PR DESCRIPTION
**Pull request by superhize**
- Can be disabled
- You can sort tems by price or items stored (both desc or asc)
- Show price from Bazaar or NPC
- Choose how much item are displayed (defaulted to 15)
- Change space between each line of text

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/config/features/Inventory.java [hunk 1]
```diff
@@ -125,18 +125,53 @@ public static class SackDisplay {
...
+                "in larger gui scale, like the nether sack.)")
+        @ConfigEditorSlider(
+                minValue = 0,
+                maxValue = 45,
+                minStep = 1
+        )
+        public int itemToShow = 15;
+
+        @Expose
+        @ConfigOption(name = "Show Price", desc = "Show price for each items in sack.")
```
Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/config/features/Inventory.java]: items -> item

File: src/main/java/at/hannibal2/skyhanni/config/features/Inventory.java [hunk 2]
```diff
@@ -125,18 +125,53 @@ public static class SackDisplay {
...
+                minStep = 1)
+        public int extraSpace = 1;
+
+        @Expose
+        @ConfigOption(name = "Sorting Type", desc = "Sorting type of items in sack.")
+        @ConfigEditorDropdown(values = {"Descending (Stored)", "Ascending (Stored)", "Descending (Price)", "Ascending (Price)"})
         public int sortingType = 0;
 
+        @Expose
+        @ConfigOption(name = "Item To Show", desc = "Choose how much items are displayed. (Some sacks have too much items to fit\n" +
```
Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/config/features/Inventory.java]: how much -> how many and much -> many

File: src/main/java/at/hannibal2/skyhanni/config/features/Inventory.java [hunk 3]
```diff
@@ -125,18 +125,53 @@ public static class SackDisplay {
...
                 "§eDefault: §72,240/2.2k\n" +
                 "§eFormatted: §72.2k/2.2k\n" +
                 "§eUnformatted: §72,240/2,200")
         @ConfigEditorDropdown(values = {"Default", "Formatted", "Unformatted"})
         public int numberFormat = 1;
 
         @Expose
-        @ConfigOption(name = "Sorting Type", desc = "Sorting type of items in sacks.")
-        @ConfigEditorDropdown(values = {"Descending", "Ascending"})
+        @ConfigOption(name = "Extra space", desc = "Space between each lines of text.")
```
Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/config/features/Inventory.java]: lines -> line

File: src/main/java/at/hannibal2/skyhanni/config/features/Inventory.java [hunk 4]
```diff
@@ -125,18 +125,53 @@ public static class SackDisplay {
         public boolean enabled = true;
 
         @Expose
-        @ConfigOption(name = "Number Format", desc = "Either show Default, Formatted or Unformatted numbers." +
+        @ConfigOption(name = "Number Format", desc = "Either show Default, Formatted or Unformatted numbers.\n" +
                 "§eDefault: §72,240/2.2k\n" +
                 "§eFormatted: §72.2k/2.2k\n" +
                 "§eUnformatted: §72,240/2,200")
```
Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/config/features/Inventory.java:131]: Should this line be 
2,240/2,240
or am i missing something?

Comment (superhize) [reply src/main/java/at/hannibal2/skyhanni/config/features/Inventory.java:131]: I can't get the exact capacity of sacks, i'm just formatting what hypixel give me (2.2k here)
- 2,240/2.2k -> thats what hypixel display
- 2.2k/2.2k -> everything is formatted, so 2,240 give 2.2k and i don't touch sack capacity
- 2,240/2,200 -> i only format sack capacity (2.2k) so it give 2,200

That's just so users can see what they gonna get in the display, i can change to 2240/2240, does not really matter

**Comment by superhize** [comment]
I'll add price in gemstones sack tomorrow

**Pull request merged**